src/components/federated-solr-faceted-search.js

```diff
@@ -84,18 +91,20 @@ class FederatedSolrFacetedSearch extends React.Component {
       ? (<ResultPendingComponent bootstrapCss={bootstrapCss} />)
       : null;
 
-    const pagination = query.pageStrategy === 'paginate' ?
-      <PaginateComponent {...this.props} bootstrapCss={bootstrapCss} onChange={onPageChange} /> :
-      null;
+    /* eslint-disable react/jsx-props-no-spreading */
```

Comment:
`props` contains a lot of properties, and I wasn't clear what they all are. I believe this is a valid case to disable the rule instead of explicitly listing all of them every time. We can revisit this later, though, if needed.